otx/algorithms/classification/adapters/mmcls/datasets/__init__.py

```diff
@@ -14,30 +14,32 @@
 # See the License for the specific language governing permissions
 # and limitations under the License.
 
-from .datasets import (
+from .otx_datasets import (
```

Comment:
I don't think that rename to otx_datasets is necessary. can we name it dataset.py to align with others?

Reply:
I renamed to avoid `datasets.dataset` and reveal otx-related dataset. Can I go as-is?